fitbenchmarking/fitbenchmarking
# update ralfit options

#### Description of Work

Fixes #1078 


#### Testing Instructions

1. check coverage hasn't decreased
2. check new options are documented and work
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
